<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>New Project</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../icons/new_project.png</normalon>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QComboBox" name="template_combo">
     <property name="placeholderText">
      <string>Project Template</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="submit_button">
     <property name="text">
      <string>Add Project</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="addtemplate_button">
     <property name="text">
      <string>Add to Templates</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTextEdit" name="details_tedit">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="tabStopWidth">
      <number>20</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>details_tedit</tabstop>
  <tabstop>template_combo</tabstop>
  <tabstop>addtemplate_button</tabstop>
  <tabstop>submit_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
